spec/posts_controller_spec.rb

```diff
@@ -0,0 +1,37 @@
+require 'rails_helper'
+
+RSpec.describe 'Posts', type: :request do
```

Comment:
- Hi, Can you create a directory called requests as it is said in the requirement(Create a Request spec file for all your controllers.)? and move your controllers test inside?
**Hint**:  `users_controller_spec.rb` and `posts_controller_spec.rb` will be in `spec/requests/` directory